docs/run/run-jobs-batch.mdx

```diff
@@ -46,3 +46,8 @@ The following example shows how you can partition a long list of circuits into m
          job = sampler.run(partitioned_circuits)
          jobs.append(job)
  ```
+
+
+<Admonition type="note">
+    If you set `backend=backend` in a v2 primitive it will run the program in job mode, even if it's inside a batch/session context. So, preferred way to use it is `sampler = SamplerV2()` when running in batch mode.
```

Comment:
I wonder if we should phrase this more strongly; what do you think of this?

```suggestion
<Admonition type="caution">
    If you set `backend=backend` in a v2 primitive it will run the program in job mode, even if it's inside a batch/session context. Avoid setting the backend parameter if you want to run in batch/session mode.
```

Reply:
I think its a good idea

Reply:
No; if you're happy with the suggestion we can commit it and merge the PR

Reply:
Sure, go ahead.